src/containers/itemcollection/index.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react'
+
+import { connect } from 'react-redux'
+
+import PropTypes from 'prop-types'
+import { Grid, Typography } from '@material-ui/core'
+
+import Item from 'components/item'
+import EditItem from 'components/edititem'
+
+import { addItem } from 'actions/item'
+
+export class ItemCollection extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      isEdit: false,
+      name: '',
+      _id: '',
+      size: 1
+    }
+
+    this.addEditItem = this.addEditItem.bind(this)
+    this.displayEditItem = this.displayEditItem.bind(this)
+    this.handleEditItemChange = this.handleEditItemChange.bind(this)
+    this.handleEditItemEnterKey = this.handleEditItemEnterKey.bind(this)
+    this.handleEditItemEscKey = this.handleEditItemEscKey.bind(this)
+  }
+
+  addEditItem () {
+    this.setState({
+      isEdit: true,
+      _id: this.uuid(),
+      name: '',
+      size: 1
+    })
+  }
+
+  uuid() {
+    var text = "i";
+    var possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
```

Comment:
can you get rid of the lowercase letters from possible?

Reply:
lower case letter for uuid?